README.example.md

```diff
@@ -77,69 +66,30 @@ stub any external calls using the test suite's features.
 Setting up the configuration is a matter of copying the supplied examples and
 filling in the gaps, as follows:
 
+1. `cp .env.example .env`
+2. `cp .env.development.example .env.development`
+3. Fill in env vars that are local to your setup
 
-## Requirements
-
-You must have the following installed and available on your machine:
-
-- **Ruby 2.6.5**
-- **Node JS 12.x**
-- **PostgreSQL 11.3**
-
-## Set up the app the first time
-```sh
-$ cp .env.example .env
-# edit .env in your editor of choice
-
-$ cp .env.development.example .env.development
-# edit .env.development in your editor of choice
-```
-
-Install the Ruby and JS dependencies
-
-```sh
-$ bundle install
-$ npm install -g yarn
-$ yarn install
-```
-
-Prepare the database
-
-You can set `DATABASE_URL` in `.env.local`, if for instance you use Docker for Postgres. `DATABASE_URL="postgresql://localhost:5432"`
```

Comment:
Please keep this.

Reply:
I removed this because under "configuration files" we also have:
> There are two main configuration files that are of concern:
> 
> - `.env` - configuration that's common across all environments that you intend
>  to run on your machine (e.g. database connection)

In hindsight I agree that it perhaps wasn't the right decision. I think I might move this to "Database setup" and reference it from the part about the configuration file.

Reply:
I've added this again, under "Configuration"